Makefile

```diff
@@ -19,3 +19,9 @@ images:
 tests:
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos6 ./run_unit_tests.sh
 	@docker run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos7 ./run_unit_tests.sh
+
+lint:
+	bash -c "[[ ! -z ./convert2rhel/ ]] && find ./convert2rhel/ -name '*.py' | sort -u | xargs pylint --rcfile=.pylintrc"
+
+lint-errors:
+	bash -c "[[ ! -z ./convert2rhel/ ]] && find ./convert2rhel/ -name '*.py' | sort -u | xargs pylint --rcfile=.pylintrc --errors-only"
```

Comment:
```suggestion
lint:
	pylint --rcfile=.pylintrc ./convert2rhel/

lint-errors:
	pylint --rcfile=.pylintrc --errors-only ./convert2rhel/
```

Reply:
Indeed that big line doesn't seem to achieve anything better than letting pylint go through the filestructure.